docs/joss/paper.md

```diff
@@ -0,0 +1,217 @@
+
+Several simple parameterizations for unresolved physical processes
+such as precipitation or the boundary layer are implemented, and new ones can
+be externally defined and passed as an argument to the model constructor.
+SpeedyWeather.jl is an intermediate-complexity general circulation model [@Kucharski2013]
+and research playground with an (almost) everything-flexible attitude.
+It can be thought of as a conceptual reinvention of the Fortran SPEEDY model [@Molteni2003]
+in the Julia programming language [@Bezanson2017].
+
+SpeedyWeather.jl internally uses three sub-modules `RingGrids`,
+`LowerTriangularMatrices`, and `SpeedyTransforms`. `RingGrids` is a module that discretizes
+the sphere on iso-latitude rings and implements interpolations between various such grids.
+`LowerTriangularMatrices` is a module used to define the spectral space of the spherical
+harmonic coefficients. `SpeedyTransforms` implements the spectral transform between
+the grid-point space as defined by `RingGrids` and the spectral space defined in
+`LowerTriangularMatrices`. These three modules are independently usable
+and therefore make SpeedyWeather.jl, beyond its main purpose of simulating 
+atmospheric motion, also a library for the analysis of gridded data on the sphere.
+Running and analyzing simulations can be interactively combined, enhancing user
+experience and productivity.
+
+The user interface of SpeedyWeather.jl is heavily influenced by
+the Julia ocean model Oceananigans.jl [@Ramadhan2020].
+A monolithic interface based on parameter files is avoided in favor of a
+library-style interface. Users write notebooks, directly into
+Julia's read-evaluate-print loop (REPL) or short scripts to run models
+rather than merely supplying parameters and input arrays.
+A model is created bottom-up by first defining the discretization
+and any non-default model components with their respective parameters.
+All components are then collected into a single model object which, once
+initialized, returns a simulation object. A simulation contains everything,
+the model with all parameters as created before but also all prognostic and diagnostic variables.
+Such a simulation can then be run, but also accessed before and after to analyze or
+visualize the current variables, or individual terms of the equations.
+One can also adjust parameters or define new model components before resuming the simulation.
+While these steps can be written into a script for reproducibility,
+the same steps can be executed and interacted with one-by-one in
+the REPL or in a single Jupyter or Pluto notebook.
+We thereby achieve an interactivity of a simulation and its various model components
+far beyond the options provided in a monolithic interface.
+At the same time, defaults, set to well-established test cases, 
+enable even inexperienced users to run simulations in just a few lines of code. 
+
+To be extensible with new model components, SpeedyWeather.jl relies on Julia's
```

Comment:
Also here I think "new model components" is a bit ambiguous. I think you can add new parameterizations easily, with the condition that similar physics (aka the model component) is already implemented and accounted for right? I would clear a bit the air by specifying: "All model components can be extended to support the custom parameterization of choice by relying on Julia's multiple dispatch programming paradigm"